lib/appium.js

```diff
@@ -344,32 +346,136 @@ class AppiumDriver extends BaseDriver {
     }
   }
 
+  pluginsToWrapCmd (cmd) {
+    return this.plugins.filter((plugin) => plugin.wrapCommands.includes(cmd));
+  }
+
+  pluginsToHandleCmd (cmd) {
+    return this.plugins.filter((plugin) => {
+      if (plugin.commands === true) {
+        return true;
+      }
+      if (_.isArray(plugin.commands) && plugin.commands.includes(cmd)) {
+        return true;
+      }
+      return false;
+    });
+  }
+
   async executeCommand (cmd, ...args) {
```

Comment:
this method is becoming too big. How about extracting parts of it into separate methods?

Reply:
I'll see what I can do about a refactor here.